tests/integration/orders-update.spec.js

```diff
@@ -40,6 +43,7 @@ test('the module should modify an existing order', (t) => {
       ),
     )
     // Create needed data
+    .then(() => ordersUpdate.client.channels.ensure(channelKey, channelRole))
```

Comment:
Yeah, I noticed there is a custom method in the SDK, that does the function of an 'upsert'(creating or updating)

So less code for us 🤓